### PR TITLE
Refs #29601: Drop extras for koji collection for Foreman 2.2+

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Related https://github.com/theforeman/foreman-packaging/pull/5085

File: koji/collection-mash-split.py [hunk 1]
```diff
@@ -385,7 +385,7 @@ def main():
             git_tag = "rpm/{}".format(version)
 
         CONFIG["comps_path"] = "/mnt/koji/mash/comps/foreman"
-        if collection == 'foreman':
+        if collection == 'foreman' and version in ['1.22', '1.23', '1.24', '2.0', '2.1']:
```
Comment (mmoll) [inline koji/collection-mash-split.py:388]:
```suggestion
        if collection == 'foreman' and version in ['1.24', '2.0', '2.1']:
```

Comment (ehelms) [reply koji/collection-mash-split.py:388]: If you look at the file, there are references back to 1.22 in multiple places. I'd rather us clean that up all at once than just here in my PR.

Comment (mmoll) [reply koji/collection-mash-split.py:388]: @ehelms please rebase, so 1f5eec57e490a25a77ecc95b34d01be31079b718 is included ;)

**Comment by ehelms** [comment]
updated @mmoll

**Pull request merged**